src/Controls/src/Build.Tasks/nuget/buildTransitive/net6.0-maccatalyst13.1/Microsoft.Maui.Controls.DefaultItems.targets

```diff
@@ -0,0 +1,36 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<!--
+	Set up some basic structuring.
+	
+	This file does two things:
+		- Move all the .xaml.cs files under their .xaml files.
+		- Remove the .xaml and .css from the @(None) item group.
+	
+	These are in the workload so they can run when the NuGet packages are not yet restored.
+	The single project system also has various globs and changes, but that does not affect
+	how the solution explorer "looks". As a result, we can leave it there for now as it is
+	useful to have in the packages for independent updates - if need be.
+	
+	This is duplicated in the workload Microsoft.Maui.Sdk.DefaultItems.targets for now because .NET MAUI
+	is also consumed via NuGet - machines that have older workloads need to also update these items.
+	-->
+	<ItemGroup Condition="'$(EnableDefaultItems)'=='True' And '$(EnableDefaultXamlItems)'=='True' And '$(EnableDefaultEmbeddedResourceItems)'=='True'">
+		<Compile Update="**\*.xaml$(DefaultLanguageSourceExtension)" DependentUpon="%(Filename)" SubType="Code" />
+		<None Remove="**\*.xaml" Condition="'$(EnableDefaultNoneItems)'=='True'" />
+	</ItemGroup>
+
+	<ItemGroup Condition="'$(EnableDefaultItems)'=='True' And '$(EnableDefaultCssItems)'=='True' And '$(EnableDefaultEmbeddedResourceItems)'=='True'">
+		<None Remove="**\*.css" Condition="'$(EnableDefaultNoneItems)'=='True'" />
```

Comment:
I'm checking if this is effectively something new, or really duplicating some other logic? (I read your comment above, but I want to be sure.). The MAUI Blazor package targets also do some funky things with CSS files and I want to make sure this doesn't affect that behavior: https://github.com/dotnet/maui/blob/main/src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets#L40-L95

Can you confirm that in a MAUI Blazor Hybrid app that CSS files are working in the default template and/or the Controls Sample, and are showing up as expected in the web view?

Reply:
<img width="1024" alt="Screenshot 2023-08-31 at 1 46 54 PM" src="https://github.com/dotnet/maui/assets/89540402/c076e2e7-cbf2-42dd-8846-730258d91d6f">

Would a css example be the razor component with red-dashed border? The Blazor sample does look the same for everything, though.


Hmm... those first two item groups in the defaultitems.target are duplicated both in the netframework2.0 default items in that Nuget folder as well as the general workload for Maui. I did place them in for consistency's sake in the MacCatalyst one as well, but I'll go ahead and remove them if there's a chance of a conflict, since I am mostly just interested in the custom entitlement.